### PR TITLE
Adding docker support for contaienr_inspect and listen to ports with docker expose

### PR DESCRIPTION
**Pull request by filipevarjao**

### CODE REVIEW

File: test/integration/docker_build_test.exs [hunk 1]
```diff
@@ -115,8 +115,32 @@ defmodule ExDockerBuild.Integration.DockerBuildTest do
...
+        {"EXPOSE", "80/tcp"}
+      ]
+
+      log =
+        capture_log(fn ->
+          assert {:ok, image_id} = DockerBuild.build(instructions, "")
+          assert {:ok, container_id} = ExDockerBuild.create_container(%{"Image" => image_id})
+          assert {:ok, container_id} == ExDockerBuild.start_container(container_id)
+          assert {:ok, container_id} == ExDockerBuild.stop_container(container_id)
+          assert {:ok, body} = ExDockerBuild.container_inspect(container_id, false)
```
Comment (sescobb27) [inline test/integration/docker_build_test.exs:136]: can you assert that the port 80 is exposed from the inspect body?

File: lib/ex_docker_build/docker_build.ex [hunk 2]
```diff
@@ -123,8 +123,10 @@ defmodule ExDockerBuild.DockerBuild do
   # defp exec({"LABEL", args}, context, _path) do
   # end
 
-  # defp exec({"EXPOSE", args}, context, _path) do
-  # end
+  defp exec({"EXPOSE", args}, context, _path) do
+    Map.merge(context, %{"Ports" => args})
```
Comment (sescobb27) [inline lib/ex_docker_build/docker_build.ex]: can you add support for `"80/tcp"` and `"80"` with tcp as default? the same as Dockerfile specification https://docs.docker.com/engine/reference/builder/#expose so we can use

```dockerfile
EXPOSE 80
EXPOSE 80/tcp
EXPOSE 80/udp
```

File: test/integration/docker_build_test.exs (hunk 1 above)
Comment (sescobb27) [reply test/integration/docker_build_test.exs:136]: also, can you add a test for `EXPOSE 80` with tcp as default

File: test/integration/docker_build_test.exs [hunk 3]
```diff
@@ -115,8 +115,32 @@ defmodule ExDockerBuild.Integration.DockerBuildTest do
...
-
       assert log =~ "STEP 1/1 : FROM alpine:latest"
     end
   end
+
+  describe "container listens on the specified network ports" do
+    test "inspecting container by container id" do
+      instructions = [
+        {"FROM", "alpine:latest"},
+        {"VOLUME", @cwd <> ":/data"},
```
Comment (sescobb27) [inline test/integration/docker_build_test.exs]: can you delete `VOLUME, RUN, CMD` steps from the instructions as they are not necessary?

File: lib/ex_docker_build/docker_build.ex (hunk 2 above)
Comment (filipevarjao) [reply lib/ex_docker_build/docker_build.ex]: Like `defp exec({"EXPOSE", args} // {"EXPOSE", "80/tcp"}, context, _path)`

Comment (filipevarjao) [reply lib/ex_docker_build/docker_build.ex]: do you mean like `defp exec({"EXPOSE", args} \\ {"EXPOSE", "80/tcp"}, context, _path) do`

Comment (sescobb27) [reply lib/ex_docker_build/docker_build.ex]: no, if i use `EXPOSE 8080` it should be used as `8080/tcp`

**Review by sescobb27 — APPROVED** [review]
LGTM 👍

**Pull request merged**